client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -59,84 +56,136 @@ import {
 	isRequestingSites,
 	getSiteAdminUrl,
 } from 'calypso/state/sites/selectors';
-import {
-	getSelectedSite,
-	getSelectedSiteId,
-	getSelectedSiteSlug,
-} from 'calypso/state/ui/selectors';
+import { getSelectedSite, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 import './style.scss';
 
 /**
  * Module variables
  */
 const SHORT_LIST_LENGTH = 6;
 const SEARCH_RESULTS_LIST_LENGTH = 12;
-const VISIBLE_CATEGORIES = [ 'new', 'popular', 'featured' ];
-const VISIBLE_CATEGORIES_FOR_RECOMMENDATIONS = [ 'new', 'popular' ];
-
-export class PluginsBrowser extends Component {
-	static propTypes = {
-		isRequestingRecommendedPlugins: PropTypes.bool.isRequired,
-		recommendedPlugins: PropTypes.arrayOf( PropTypes.object ),
-		selectedSite: PropTypes.object,
-		trackPageView: PropTypes.bool,
-		hideHeader: PropTypes.bool,
-	};
 
-	static defaultProps = {
-		trackPageViews: true,
-	};
-
-	constructor( props ) {
-		super( props );
-		this.state = {
-			isMobile: isWithinBreakpoint( '<960px' ),
-		};
-	}
+export const PluginsBrowser = ( props ) => {
```

Comment:
`export` can probably be removed, since we are exporting this on the bottom of the file.

Reply:
The export on the bottom is wrapped by `UrlSearch` while this exports only the component. 

In this case, where our unit tests are mounting the component it doesn't make any difference, but it can make if we use the shallow render.

Reply:
Thanks for the explanation!